tock/api/views.py

```diff
@@ -54,7 +54,7 @@ class UserDataSerializer(serializers.Serializer):
     organization = serializers.StringRelatedField()
 
     def get_unit(self,obj):
```

Comment:
I think there's an opportunity here to drop this altogether and let the `StringRelatedField` handle the work like we're doing for `organization`

Reply:
Initially I had some concerns about scope creep, but after searching I don't see where we actually call `get_unit` anywhere, so may as well drop it.

Reply:
Correction... without this one we throw a serializer error that looks weird enough that I don't want to track it down for this story. Let's log a separate issue